src/utilities.cpp

```diff
@@ -136,7 +136,8 @@ void renameFile(FmFileInfo *file, QWidget *parent) {
 
   GFile* gf = fm_path_to_gfile(path);
   GFile* parent_gf = g_file_get_parent(gf);
-  GFile* dest = g_file_get_child(G_FILE(parent_gf), new_name.toLocal8Bit().data());
```

Comment:
What's the problem here? Doesn't the `g_file_get_child` make a copy of the received string if it needs/stores it in the `GFile`? If so, then there is no problem with the original call as the string/pointer is valid throughout the whole call of the `g_file_get_child`..